impala/__init__.py

```diff
@@ -14,6 +14,4 @@
 
 from __future__ import absolute_import
 
-from ._version import get_versions
-__version__ = get_versions()['version']
-del get_versions
+__version__ = u'v0.20.0'
```

Comment:
Should this match the version in setup.py?

Reply:
Ouch, yes, probably the main reason to use versioneer is to avoid these kind of mistakes :)
https://packaging.python.org/en/latest/discussions/single-source-version/

This could be trouble if it was in an inconsistent state when a release is created, though generally __version__ is not used when a library is imported.

Reply:
btw fixed it to use v0.20.0 is both cases
will have to bump both when preparing a release

Reply:
Let's add a code comment to remind people to update the other location as well. For both spots.

Reply:
good idea, done